packages/website/docs/docs/side-menu.md

```diff
@@ -94,21 +159,28 @@ export default function App() {
 
 `CustomDragHandleMenu` is the component we use to replace the default Drag Handle Menu. You can see it's made up of a bunch of other components that are exported by BlockNote. Read on to [Components](/docs/side-menu#components) to find out more about these.
 
+Here, we're passing it as a prop to the `DefaultSideMenu` component, but you can also pass it to the `DragHandle` component we saw in the [previous demo](/docs/side-menu#custom-side-menu), if you don't want to use the default Side Menu layout.
+
 After creating `CustomDragHandleMenu`, we tell BlockNote to use it inside `BlockNoteView`. [Changing UI Elements](/docs/ui-elements) has more information about how this is done.
 
 ## Components
 
-It might seem daunting to create your own Formatting Toolbar from scratch, which is why BlockNote provides React components that you can use which match the default styling.
+It might seem daunting to create your own Side Menu or Drag Handle Menu from scratch, which is why BlockNote provides React components that you can use which match the default styling.
```

Comment:
personal take, think this paragraph is a bit too much. I would focus on the core and just make it

> BlockNote provides React components that you can reuse to easily create your menus.